agave_app/ViewerState.cpp

```diff
@@ -299,6 +308,21 @@ stateToLoadSpec(const Serialize::ViewerState& state)
   return spec;
 }
 
+ColorRamp
+stateToColorRamp(const Serialize::ViewerState& state, int channelIndex)
```

Comment:
this is the "input" from file to app.  taking the Serialize::ViewerState structure and translating it to the app internals.